packages/cloud_firestore/cloud_firestore_web/lib/cloud_firestore_web.dart

```diff
@@ -133,6 +133,14 @@ class FirebaseFirestoreWeb extends FirebaseFirestorePlatform {
   /// Enable persistence of Firestore data.
   @override
   Future<void> enablePersistence([PersistenceSettings? settings]) {
+    if (settings != null) {
+      firestore_interop.PersistenceSettings interopSettings =
+          firestore_interop.PersistenceSettings(
+              synchronizeTabs: settings.synchronizeTabs);
```

Comment:
Are we missing https://firebase.google.com/docs/reference/js/firebase.firestore.PersistenceSettings#experimentalforceowningtab ?

Reply:
Yeah, see description. Happy to implement but it is experimental so wasn't sure if we should at this moment in time.